### ISSUE TITLE
Conversion of DONUT model to TensorRT

### ISSUE THREAD

**Issue by mfournioux** [triton-inference-server/server#6734]
**Description**

I am trying to convert Donut model on tensorRT backend.
Here is some documentation about this model :
https://huggingface.co/docs/transformers/model_doc/donut

**Triton Information**

Are you using the Triton container or did you build it yourself?
I am using for the conversion the following image : nvidia/tensorrt:23.11-py3

**To Reproduce**

**Step 1 : Convert the model from HuggingFace to onnx format :** 

I am using the optimum library from HF.

optimum-cli export onnx --model ../data/ tmp/ --task="image-to-text" --monolith

**Step 2 : Convert the model from onnx format to tensorrt:**

I am using the cli : trtexec.

trtexec --onnx=tmp/model.onnx --saveEngine=tmp/model_engine.plan --minShapes=\'pixel_values\':1x3x2560x1920,\'decoder_input_ids\':1x8 --optShapes=\'pixel_values\':2x3x2560x1920,\'decoder_input_ids\':2x8 --maxShapes=\'pixel_values\':8x3x2560x1920,\'decoder_input_ids\':8x8 --useCudaGraph

**Step 3 : test to infer the tensorRT engine created :** 

trtexec --loadEngine=/workspace/tmp/model_engine.plan --useCudaGraph --loadInputs=pixel_values:/workspace/pixel_values.dat,decoder_input_ids:/workspace/decoder_input_ids.dat --exportOutput=/workspace/results.json.

**Expected behavior**
When I try to do the step 3, the cli trtexec only provides the logits. I have tried to post process them with an argmax operation and then decode to get the text, but the text output is not coherent to what I expected. 

I have two questions : 
1. Is it adviced to convert DONUT model to tensorRT with the cli trtexec or is it better to follow this procedure https://github.com/NVIDIA/TensorRT/tree/release/9.2/demo/HuggingFace/Vision2Seq ?
2. Why the engine created only gives back the logits without any post processing on logits like argmax for instance?

Many thanks for your help

**Comment by Tabrizian** [comment]
It sounds like a question for the TensorRT team. Please post this question on the TensorRT or https://forums.developer.nvidia.com and folks with more relevant expertise might be able to help you.
